vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -61,6 +62,10 @@ def get_endpoints(inventory)
       end
     end
 
+    def get_namespaces(inventory)
+      process_collection(inventory["namespace"], :container_projects) { |n| parse_namespaces(n) }
```

Comment:
I thought I had commented about adding also (maybe I haven't submitted it):

    @data[:container_projects].each do |ns|
        @data_index.store_path(:container_projects, :by_name, ns[:name], ns)
    end